### PR TITLE
Change figure numbering

### LINKED ISSUE #184: Figure numbering consistency/update
Update needed to current figure number approach, as we will currently end up with figure numbers in the 100's (note how the seasonal forecast NB starts from 4, due to the three figures in Methane).

For climate projections they manually set the figure labels (Fig A etc), but I think that might also get messy (and the syntax is different).

My preference would be some way to reset the numbering per section or per assessment, but that will lead to labelling conflicts. Possibly this is okay if the labels are not used.

Once we decide, the template needs to be update!

### PR DESCRIPTION
**Pull request by malmans2**
Closes #184

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -30,27 +31,32 @@ parts:
...
     - glob: Reanalyses/*
 
 - caption: ➡️ Seasonal Forecasts
+  numbered: 2
   chapters:
   - file: Seasonal_Forecasts/seasonal.md
     title: Quality assessments for Seasonal Forecasts
     sections:
-    - glob: Seasonal_Forecasts/*
+    - file: Seasonal_Forecasts/seasonal_seasonal-monthly-single-levels_forecast-skill_q02.ipynb
```
Comment (crgoddard) [inline _toc.yml:55]: why does the glob need to be removed here?

File: _toc.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ root: intro
 parts:
 
 - caption: ➡️ Satellite Observations
+  numbered: 3
```
Comment (crgoddard) [inline _toc.yml:9]: Checking with others if we like this, things start to look a bit strange with 4 levels in the figure numbers.

**Review by crgoddard — COMMENTED** [review]
One question on the seasonal toc, other than that, checking with others if we like the change.

File: _toc.yml (hunk 1 above)
Comment (malmans2) [reply _toc.yml:55]: sorry my bad, I was trying various things and I forgot to restore the original

File: _toc.yml (hunk 2 above)
Comment (malmans2) [reply _toc.yml:9]: One option is to remove one level. So Satellite Observations shows up in the part caption, then we use flat chapters:
```
- caption: ➡️ Satellite Observations
  numbered: 2
  chapters:
  - file: Satellite_ECVs/Atmosphere_Physics/Atmosphere_Physics.md
    sections:
    - glob: Satellite_ECVs/Atmosphere_Physics/*
  - file: Satellite_ECVs/Atmospheric_Composition/Atmospheric_Composition.md
    sections:
    - glob: Satellite_ECVs/Atmospheric_Composition/*
  - file: Satellite_ECVs/Cryosphere/Cryosphere.md
    sections:
    - glob: Satellite_ECVs/Cryosphere/*
  - file: Satellite_ECVs/Land_Biosphere/Land_Biosphere.md
    sections:
    - glob: Satellite_ECVs/Land_Biosphere/*
  - file: Satellite_ECVs/Land_Hydrology/Land_Hydrology.md
    sections:
    - glob: Satellite_ECVs/Land_Hydrology/*
  - file: Satellite_ECVs/Ocean/Ocean.md
    sections:
    - glob: Satellite_ECVs/Ocean/*
```

**Pull request merged**